server/plugin/command.go

```diff
@@ -600,12 +600,18 @@ func getAutocompleteData(config *Configuration) *model.AutocompleteData {
 	subscriptionsAdd.AddTextArgument("Owner/repo to subscribe to", "[owner/repo]", "")
 	subscriptionsAdd.AddTextArgument("Comma-delimited list of one or more of: issues, pulls, pulls_merged, pushes, creates, deletes, issue_creations, issue_comments, pull_reviews, label:\"<labelname>\". Defaults to pulls,issues,creates,deletes", "[features] (optional)", `/[^,-\s]+(,[^,-\s]+)*/`)
 	if config.GitHubOrg != "" {
-		flags := []model.AutocompleteListItem{{
-			HelpText: "Events triggered by organization members will not be delivered (the organization config should be set, otherwise this flag has not effect)",
-			Hint:     "(optional)",
-			Item:     "--exclude-org-member",
-		}}
-		subscriptionsAdd.AddStaticListArgument("Currently supports --exclude-org-member", false, flags)
+		flags := []model.AutocompleteListItem{
+			{
+				HelpText: "Events triggered by organization members will not be delivered (the organization config should be set, otherwise this flag has not effect)",
+				Hint:     "(optional)",
+				Item:     "--exclude-org-member",
+			},
+			{
+				HelpText: "Notifications will be delievered with titles only (for example, the body of a pull request will not be displayed)",
+				Hint:     "(optional)",
+				Item:     "--title-only",
+			}}
+		subscriptionsAdd.AddStaticListArgument("Currently supports --exclude-org-member and --title-only", false, flags)
```

Comment:
In the state of this PR, we are not able to use both the `exclude-org-member` _and_ `title-only` flags simultaneously. This is because they are being treated as a single positional argument, each showing up as a single selectable option in the same argument.

Similar to what was discussed in https://github.com/mattermost/mattermost-plugin-github/pull/505#discussion_r796253955, we should use "named" arguments with explicit `true/false` values, rather than using positional arguments that implicitly mean `true`.

Also, this `title-only` flag should be available even when `config.GitHubOrg` is not defined, but it's currently added in this if block which restricts to when `config.GitHubOrg` is defined.